File: js/const-enum.ts
// 由关键字 ```const``` 声明常量
// 由关键字 ```enum``` 枚举类型有数字枚举、字符串枚举，可以手动指定原始值

//

// 枚举
// 默认情况下，从 0 开始为元素编号
// 可以手动指定原始值
enum Color {
    read,
    green,
    blue
}
console.log(Color.blue); // 结果： 2

// 常量
const uid: number = 10;
//uid = 11; // error

File: golang/const_enum.go
package main

// 由关键字```const```声明常量，没有枚举类型

//

// 定义常量并使用 iota 模拟枚举
const (
	read int = iota
	green
	blue
)

const uid int = 10

// uid =11; // error
